src/ensembl/webpack/webpack.config.dev.js

```diff
@@ -68,9 +68,13 @@ const devConfig = {
     // this should prevent 404 errors that usually occur in SPA on reloads
     historyApiFallback: true,
 
+    host: '0.0.0.0',
+
     // enable hot module reloading
     hot: true,
 
+    port: 4000,
```

Comment:
I've left this disabled by default along with a comment, so that if someone comes across the same issue they only need to change the boolean value from `false` to `true`.

Reply:
I have left the host as it is and removed the port so that it uses the default one.